tests/testthat/test-object-package.R

```diff
@@ -17,3 +17,94 @@ test_that("person turned into meaningful text", {
     person_desc(comment = c("ORCID" = "1234", "extra"))
   })
 })
+
+test_that("package description not affected if no links", {
+  text <- "A simple description with no links."
+
+  parsed <- package_description_urls(text)
+
+  expect_equal(
+    parsed,
+    text
+  )
+  expect_snapshot(parsed)
+})
+
+test_that("can autolink urls on package Description", {
+  urls <- paste(
+    "<https://github.com/>",
+    "Secured <https://github.com/>.",
+    "No link <www.github.com/>.",
+    "url masked with spaces",
+    "<https://database.ich.org/sites/default/files/Q1E%20Guideline.pdf>",
+    "url fully masked",
+    "<https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=OJ%3AL%3A2015%3A012%3ATOC>"
+  )
+
+  parsed <- package_description_urls(urls)
```

Comment:
This would be easier to understand if you put the input and output next to each other, e.g.

```R
expect_equal(
  package_description_urls("<https://github.com/>"),
  "\\url{https://github.com/}"
)
```